### PR TITLE
xiaoyan blog 4

### PR DESCRIPTION
**Pull request by Elena-Baixy**

### CODE REVIEW

**Comment by harvey-fin** [comment]
Great progress. For the two metrics (logit/rank difference and appearance rate), they can have very different narratives, which is what we observed in the subliminal learning project (we were using log probs). This is mainly because the prompt/prefix has a very large impact on logits and there's no standardized prompt to use.

**Comment by Elena-Baixy** [comment]
I am going to use late days for the paper draft.
